src/SFA.DAS.EmployerFinance.Jobs/ScheduledJobs/DummyJob.cs

```diff
@@ -15,13 +16,13 @@ public DummyJob(IMessageSession messageSession)
             _messageSession = messageSession;
         }
 
-        public Task Run([TimerTrigger("0 */1 * * * *", RunOnStartup = true)] TimerInfo timer, ILogger logger)
+        public async Task Run([TimerTrigger("*/2 * * * * *", RunOnStartup = true)] TimerInfo timer, ILogger logger)
```

Comment:
This job can be removed if you prefer, we'll verify that any communication is working as part of the health check ticket.